.github/ISSUE_TEMPLATE/NEW_RELEASE.md

```diff
@@ -10,14 +10,17 @@ assignees: ahg-g, alculquicondor, ArangoGutierrez, denkensk
 <!--
 Please do not remove items from the checklist
 -->
-- [ ] All [OWNERS](https://github.com/kubernetes-sigs/kueue/blob/main/OWNERS) must LGTM the release proposal
-- [ ] Verify that the changelog in this issue is up-to-date
+- [ ] [OWNERS](https://github.com/kubernetes-sigs/kueue/blob/main/OWNERS) must LGTM the release proposal.
```

Comment:
@alculquicondor this was causing the failure but not sure if I did something wrong..